app/javascript/packages/document-capture/components/document-capture-review-issues.tsx

```diff
@@ -28,12 +28,8 @@ interface DocumentCaptureReviewIssuesProps {
   hasDismissed: boolean;
 }
 
-type DocumentSide = 'front' | 'back';
+type DocumentSide = 'front' | 'back' | 'selfie';
 
-/**
- * Sides of the document to present as file input.
- */
-const DOCUMENT_SIDES: DocumentSide[] = ['front', 'back'];
```

Comment:
just checking my own understanding: are we removing the `DOCUMENT_SIDES` var and replacing it with `documentSides` because the full caps version denotes a `constant`, when using the keyword definition of `const` already does that?

Reply:
This is a great question, and there's a lot of layers. I'm going to use a pullquote from [here](https://mariusschulz.com/blog/constant-variables-in-javascript-or-when-const-isnt-constant) to try and explain.

> In JavaScript, const does not mean constant, but one-time assignment.

In the original code, `const DOCUMENT_SIDES` can only ever be this array `['front', 'back']`. This is what `constant` really means. It doesn't matter what you do in the code, you can depend that `DOCUMENT_SIDES` will always be the same.

In the new code `const documentSides` can be two different things, depending on the situation. This is 'one time assignment' but it's not really constant.

So to try and sum that up `AN_ALL_CAPS_VARIABLE` means: This will always have the same value in all situations. Where a `const` means "We're not going to reassign this later in the code".

Example of reassignment
```javascript
const documentSides = ['front', 'back']
// 👇 This code will fail, because I'm reassigning.
documentSides = ['somethingElse']
```

Reply:
Ohhhh this is a really helpful explanation (and link)! Thank you so much. I understand now.